memorywipe.py

```diff
@@ -0,0 +1,496 @@
+        strongp = click.prompt("So set a random password with special characters: ", hide_input=True, confirmation_prompt=True)
+        out = subprocess.run(["sudo", "veracrypt", "-tc", f"--volume-type={vol_type}", 
+                                f"{self.partition}", f'--encryption={encrypt_type}', f'--hash={hash_algo}', 
+                                f"--filesystem={filesystem}", "-p", f"{strongp}", "--pim=0", 
+                                "-k", "", "--random-source=/dev/urandom"], capture_output=True)
+        if out.returncode:
+            click.secho(out.stderr.decode(), fg="yellow")
+        show_out = click.prompt("Do you wish to see the output of the command run?", type=click.Choice(["y", "n"]), default="n")
+        if show_out == "y":
+            click.echo(out.stdout.decode())
+                  
+    def _veracrypt_encrypt(self):
+        if self.chk_encrypted():
+            click.echo("Selected partition is already encrypted! Skipping encryption...\n")
+        else:
+            q2 = click.prompt("Do you want to encrypt manually or automatically?", type=click.Choice(["m", "a"]), show_choices=True, default="a")
+            if q2 == "m":
+                click.echo("Starting veracrypt manually...\n")
+                self._veracrypt_interactive()
+                click.secho(f"Finished encrypting {self.partition}\n", fg="green")
+            elif q2 == "a":
+                click.echo("Uses AES, with SHA-512 and makes an NTFS filesystem")
+                click.echo("\nEnter strong password for encrypting. (You don't have to remember it)")
+                strongp = click.prompt("So set a random password with special characters: ", hide_input=True, confirmation_prompt=True)
+                out = subprocess.run(["sudo", "veracrypt", "-tc", "--volume-type=normal", 
+                                f"{self.partition}", "--encryption=aes", "--hash=sha-512", 
+                                "--filesystem=ntfs", "-p", f"{strongp}", "--pim=0", 
+                                "-k", '""', "--random-source=/dev/urandom"], capture_output=True)
+                click.secho(f"Finished encrypting {self.partition}\n", fg="green")
+                              
+    def _wipe_disk(self):
+        click.echo(f"Wiping {self.partition}...")
+        if not self._check_mount():
+            click.secho("Make sure the disk is unmounted!", fg="red")
+            raise click.Abort
+        else:
+            subprocess.run(["sudo", "dd", "if=/dev/random", f"of={self.partition}", "bs=1M", "status=progress"])
+            subprocess.run(["sudo", "dd", "if=/dev/zero", f"of={self.partition}", "bs=1M", "status=progress"])
+            
+            click.secho(f"{self.partition} overwritten with one write each of random data and zeroes", fg="bright_white")
+            self.name = click.prompt("Set device name")
+            subprocess.run("sudo", "mkfs.ntfs", "-L", f"{self.name}", f"{self.partition}")
```

Comment:
This line is causing a type error after setting device name. Please check once.  
This is the traceback:
```
Set device name: test
Traceback (most recent call last):
  File "/home/sherl0ck/memorywipe.py", line 496, in <module>
    main()
  File "/usr/lib/python3/dist-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sherl0ck/memorywipe.py", line 435, in sanitize
    wipe.auto_wipe()
  File "/home/sherl0ck/memorywipe.py", line 400, in auto_wipe
    self.crypt_wipe()
  File "/home/sherl0ck/memorywipe.py", line 62, in crypt_wipe
    self._wipe_disk()
  File "/home/sherl0ck/memorywipe.py", line 182, in _wipe_disk
    subprocess.run("sudo", "mkfs.ntfs", "-L", f"{self.name}", f"{self.partition}")
  File "/usr/lib/python3.11/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/subprocess.py", line 835, in __init__
    raise TypeError("bufsize must be an integer")
TypeError: bufsize must be an integer
```